src/lib389/lib389/cli_conf/backend.py

```diff
@@ -241,7 +241,7 @@ def backend_import(inst, basedn, log, args):
     task = mc.import_ldif(ldifs=args.ldifs, chunk_size=args.chunks_size, encrypted=args.encrypted,
                           gen_uniq_id=args.gen_uniq_id, only_core=args.only_core, include_suffixes=args.include_suffixes,
                           exclude_suffixes=args.exclude_suffixes)
-    task.wait()
+    task.wait(timeout=None)
```

Comment:
I was thinking, maybe, we should log at least something here if the user SIGINT at that point.
(we can overwrite signal_handler here, I think)

Reply:
When you say sigint you mean they want to cancel the task? Or you want the sigint to show that the task is still waiting?

Reply:
> When you say sigint you mean they want to cancel the task? Or you want the sigint to show that the task is still waiting?

I think just 'log that the task is waiting' will be enough.
But we can go even further and ask user a question like: 'Do you want to cancel task? (Yes - Cancel, No - Keep it running in the background)'.
I am good with both options :)

Reply:
It's a cool idea. I think maybe a more intutive way is on the first sigint (ctrl-c) we print the current status of the task and if they press ctrl-c again within 1 second then we cancel?

Reply:
> It's a cool idea. I think maybe a more intutive way is on the first sigint (ctrl-c) we print the current status of the task and if they press ctrl-c again within 1 second then we cancel?

I really like this idea, although you can only get so much information from the task entry.  Regardless think this would be a great enhancement, that being said perhaps it should be done in a different ticket with a wider scope.

Reply:
Yep, that sounds reasonable. In that case it's okay to merge this as is and I'll create a new ticket for the interupt handling?

Reply:
Agree. Let's do this in a separate ticket. For this one, maybe you can log at least something but it's up to you.
You have my ack.